src/growth-pandora.R

```diff
@@ -296,7 +296,7 @@ if (nrow(FuelTypeCrosswalk) > 0) {
 useWindGrid <- nrow(WindGrid) > 0
 
 # Decide whether or not to manually set grass fuel loading and curing
-setFuelLoad <- nrow(FuelLoad) > 0
+setFuelLoad <- FALSE #nrow(FuelLoad) > 0 # TODO: Notify user that fuel loading is not used
```

Comment:
@shreeramsenthi if Fuel Loading is never used, should we remove the option from the SyncroSim datasheet?